### PR TITLE
DM: update the description about error handle command's enhancements

### PR DESCRIPTION
**Pull request by WizardXiao**
### What is changed, added or deleted? (Required)
DM adds or updates commands about `binlog`, `binlog-schema`, `shard-ddl-lock`, `config`, they will replace `handle-error`, `operate-schema`, `unlock-ddl-lock`, `show-ddl-locks`, `get-config` to make commands be more accessible.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): [#4017](https://github.com/pingcap/tiflow/pull/4017)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/handle-failed-ddl-statements.md [hunk 1]
```diff
@@ -8,78 +8,110 @@ aliases: ['/docs-cn/tidb-data-migration/dev/skip-or-replace-abnormal-sql-stateme
...
+  -h, --help                help for binlog
 
 Global Flags:
-  -s, --source strings   MySQL Source ID
+  -s, --source strings   MySQL Source ID.
+
+Use "dmctl binlog [command] --help" for more information about a command.
 ```
 
 ### 参数解释
```
Comment (ran-huang) [inline dm/handle-failed-ddl-statements.md:68]: 这里有一些是 sub-command，有些是 flag，能否分开来写？

File: dm/handle-failed-ddl-statements.md [hunk 2]
```diff
@@ -414,9 +446,28 @@ ALTER TABLE `shard_db_*`.`shard_table_*` CHARACTER SET LATIN1 COLLATE LATIN1_DAN
 
     可以看到任务运行正常，无错误信息。四条 DDL 全部被跳过。
 
-### 迁移中断执行替代操作
+#### binlog replace 命令
```
Comment (ran-huang) [inline dm/handle-failed-ddl-statements.md:449]:
```suggestion
### binlog replace 命令
```

File: dm/dm-faq.md [hunk 3]
```diff
@@ -178,7 +178,7 @@ if the DDL is not needed, you can use a filter rule with \"*\" schema-pattern to
 
 出现报错的原因是 TiDB parser 无法解析上游的 DDL，例如 `ALTER EVENT`，所以 `sql-skip` 不会按预期生效。可以在任务配置文件中添加 [Binlog 过滤规则](/dm/dm-key-features.md#binlog-event-filter)进行过滤，并设置 `schema-pattern: "*"`。从 DM 2.0.1 版本开始，已预设过滤了 `EVENT` 相关语句。
 
-在 DM v2.0 版本之后 `sql-skip` 已经被 `handle-error` 替代，`handle-error` 可以跳过该类错误。
+在 DM v2.0 版本之后 `sql-skip` 已经被 `handle-error` 替代，`handle-error` 可以跳过该类错误。在 DM v6.0 版本之后 `handle-error` 已经被 `binlog` 替代, `binlog` 可以跳过该类错误。
```
[the remark this answers is not in the document]
Comment (WizardXiao) [reply dm/dm-faq.md:181]: 不太清楚是否还有用户在使用 `sql-skip` 这个命令了，也可以直接改成 v6.0 版本的描述，跳过 v2.0 的描述？

File: dm/handle-failed-ddl-statements.md (hunk 1 above)
Comment (WizardXiao) [reply dm/handle-failed-ddl-statements.md:68]: 嗯，可以

File: dm/dm-faq.md [hunk 4]
```diff
@@ -178,7 +178,7 @@ if the DDL is not needed, you can use a filter rule with \"*\" schema-pattern to
 
 出现报错的原因是 TiDB parser 无法解析上游的 DDL，例如 `ALTER EVENT`，所以 `sql-skip` 不会按预期生效。可以在任务配置文件中添加 [Binlog 过滤规则](/dm/dm-key-features.md#binlog-event-filter)进行过滤，并设置 `schema-pattern: "*"`。从 DM 2.0.1 版本开始，已预设过滤了 `EVENT` 相关语句。
 
-在 DM v2.0 版本之后 `sql-skip` 已经被 `handle-error` 替代，`handle-error` 可以跳过该类错误。
+在 DM v6.0 版本之后 `sql-skip` 已经被 `binlog` 替代, `binlog` 可以跳过该类错误。
```
Comment (lichunzhu) [inline dm/dm-faq.md:181]: 在 DM v6.0 版本之后 `sql-skip`, `handle-error` 均已经被 `binlog` 替代, `binlog` 可以跳过该类错误。

File: dm/dm-manage-schema.md [hunk 5]
```diff
@@ -151,8 +177,24 @@ operate-schema set -s mysql-replica-01 task_single -d db_single -t t1 db_single.
     ]
 }
 ```
+### binlog-schema delete 命令
+```bash
```
Comment (lichunzhu) [inline dm/dm-manage-schema.md:181]:
```suggestion
### binlog-schema delete 命令

```bash
```

**Comment by ran-huang** [comment]
@WizardXiao PTAL [a91e9f4](https://github.com/pingcap/docs-cn/pull/8683/commits/a91e9f4d9e79a73cd43e77e2d1291ba5346589a4)

**Comment by WizardXiao** [comment]
> @WizardXiao 不支持太多层级的标题，所以调整了一下结构，PTAL [957d6d4](https://github.com/pingcap/docs-cn/pull/8683/commits/957d6d4dd828beab9a2c26ee65ac51fccd9c5749)

好的，层级更清楚了

**Review by lichunzhu — APPROVED** [review]
LGTM

File: dm/manually-handling-sharding-ddl-locks.md [hunk 6]
```diff
@@ -209,7 +249,7 @@ MySQL 及 DM 操作与处理流程如下：
 4. 由于业务需要，`mysql-replica-02` 对应的数据不再需要迁移到下游 TiDB，对 `mysql-replica-02` 执行了移除操作。
 5. DM-master 上 ID 为 ```test-`shard_db`.`shard_table` ``` 的 lock 无法等到 `mysql-replica-02` 的 DDL 操作信息。
 
-    `show-ddl-locks` 返回的 `unsynced` 中一直包含 `mysql-replica-02` 的信息。
+    `shard-ddl-lock` 返回的 `unsynced` 中一直包含 `mysql-replica-02` 的信息。
 
 6. 使用 `unlock-dll-lock` 来请求 DM-master 主动 unlock 该 DDL lock。
```
Comment (lichunzhu) [inline dm/manually-handling-sharding-ddl-locks.md:254]:
```suggestion
6. 使用 `unlock-ddl-lock` 来请求 DM-master 主动 unlock 该 DDL lock。
```

File: dm/manually-handling-sharding-ddl-locks.md [hunk 7]
```diff
@@ -231,7 +271,7 @@ MySQL 及 DM 操作与处理流程如下：
 7. 使用 `show-dd-locks` 确认 DDL lock 是否被成功 unlock。
```
Comment (lichunzhu) [inline dm/manually-handling-sharding-ddl-locks.md:271]:
```suggestion
7. 使用 `shard-ddl-lock` 确认 DDL lock 是否被成功 unlock。```

File: migrate-with-more-columns-downstream.md [hunk 8]
```diff
@@ -54,7 +54,7 @@ CREATE TABLE `messages` (
 
 当 DM 尝试使用下游表结构解析上游产生的 binlog event 时，DM 会报出上述 `Column count doesn't match` 错误。
 
-此时，你可以使用 `operate-schema` 命令来为数据源中需要迁移的表指定表结构，表结构需要对应 DM 将要开始同步的 binlog event 的数据。如果你在进行分表合并的数据迁移，那么需要为每个分表按照如下步骤在 DM 中设置用于解析 binlog event 的表结构。具体操作为：
+此时，你可以使用 `binlog-schema` 命令来为数据源中需要迁移的表指定表结构，表结构需要对应 DM 将要开始同步的 binlog event 的数据。如果你在进行分表合并的数据迁移，那么需要为每个分表按照如下步骤在 DM 中设置用于解析 binlog event 的表结构。具体操作为：
 
 1. 在 DM 中，新建一个 `.sql` 文件，并将上游表结构对应的 `CREATE TABLE` 语句添加到该文件。例如，将以下表结构保存到 `log.messages.sql` 中。
```
Comment (sunzhaoyang) [inline migrate-with-more-columns-downstream.md:59]: 这里为什么不直接用 --source/--target ？新特性更方便呀

**Review by sunzhaoyang — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

File: dm/dm-faq.md [hunk 9]
```diff
@@ -178,7 +178,7 @@ if the DDL is not needed, you can use a filter rule with \"*\" schema-pattern to
 
 出现报错的原因是 TiDB parser 无法解析上游的 DDL，例如 `ALTER EVENT`，所以 `sql-skip` 不会按预期生效。可以在任务配置文件中添加 [Binlog 过滤规则](/dm/dm-key-features.md#binlog-event-filter)进行过滤，并设置 `schema-pattern: "*"`。从 DM 2.0.1 版本开始，已预设过滤了 `EVENT` 相关语句。
 
-在 DM v2.0 版本之后 `sql-skip` 已经被 `handle-error` 替代，`handle-error` 可以跳过该类错误。
+在 DM v6.0 版本之后 `sql-skip`、`handle-error` 均已经被 `binlog` 替代，`binlog` 可以跳过该类错误。
```
Comment (ran-huang) [inline dm/dm-faq.md:181]:
```suggestion
在 DM v6.0 版本之后 `sql-skip`、`handle-error` 均已经被 `binlog` 替代，使用 `binlog` 命令可以跳过该类错误。
```

File: migrate-with-more-columns-downstream.md [hunk 10]
```diff
@@ -66,30 +66,32 @@ CREATE TABLE `messages` (
...
-    | operate-schema set| 手动设置 schema 信息 |
-    | -s             | 指定 source。`${source-id}` 表示 MySQL 数据源 ID。`${task-name}` 表示数据同步任务配置文件 `task.yaml` 中定义的同步任务名称。 |
-    | -d             | 指定 database。`${database-name}`表示上游数据库名。 |
-    | -t             | 指定 table。`${table-name}` 表示上游数据表名。`${schema-file}` 表示将被设置的表结构文件。 |
-
+    | binlog-schema update| 手动更新 schema 信息 |
+    | -s             | 指定 source。`${source-id}` 表示 MySQL 数据源 ID。 |
+    | `${task-name}` | 指定 task。表示数据同步任务配置文件 `task.yaml` 中定义的同步任务名称。|
+    | `${database-name}` | 指定 database。表示上游数据库名。 |
+    | `${table-name}` | 指定 table。表示上游数据表名。表示将被设置的表结构文件。 |
```
Comment (ran-huang) [inline migrate-with-more-columns-downstream.md:86]: Will fix it later.

**Pull request merged**